src/components/GeoMap.js

```diff
@@ -434,32 +434,43 @@ export default class GeoMap {
       let minValue = attrValue.min;
       let maxValue = attrValue.max;
 
-      if (attrValue.min === 'minValue') {
```

Comment:
You should not remove these lines. Here's their application: https://github.com/Packet-Clearing-House/maptable#scaledvalue